### PR TITLE
grounditems: highlight herblore / prayer ground items

### PR DESCRIPTION
**Pull request by zeruth**
Highlights items that could provide you with Herblore / Prayer XP.
Value is still prioritized over XP.

This includes:
```
Herblore:

Grimy Herbs
Clean Herbs
Secondary ingredients
Jungle Potion herbs
Herb seeds
Secondary seeds

Prayer:

Bones
Ensouled heads
```

All noted items should count as well.

Below is an example
image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 1]
```diff
@@ -134,6 +134,14 @@
 	private static final int CAST_ON_ITEM = MenuOpcode.SPELL_CAST_ON_GROUND_ITEM.getId();
 	private static final String TELEGRAB_TEXT = ColorUtil.wrapWithColorTag("Telekinetic Grab", Color.GREEN) + ColorUtil.prependColorTag(" -> ", Color.WHITE);
 	private final Map<Integer, Color> priceChecks = new LinkedHashMap<>();
+	private final Queue<Integer> droppedItemQueue = EvictingQueue.create(16); // recently dropped items
+	public boolean highlightHerblore;
+	public boolean highlightPrayer;
+	public LoadingCache<String, Boolean> hiddenItems;
+	Color herbloreColor;
+	Color prayerColor;
+	ArrayList<Integer> herbloreItems = new ArrayList();
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: List<Integer>

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 2]
```diff
@@ -134,6 +134,14 @@
...
+	ArrayList<Integer> prayerItems = new ArrayList();
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: List<Integer>

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 3]
```diff
@@ -981,6 +1006,255 @@ private void notifyHighlightedItem(GroundItem item)
 		notifier.notify(notificationStringBuilder.toString());
 	}
 
+
+	void createHerbloreItems()
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: A static set would probably be better for this.

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 4]
```diff
@@ -981,6 +1006,255 @@ private void notifyHighlightedItem(GroundItem item)
...
+		//Secondary seeds
+		herbloreItems.add(ItemID.LIMPWURT_SEED);
+		herbloreItems.add(ItemID.SNAPE_GRASS_SEED);
+		herbloreItems.add(ItemID.POTATO_CACTUS_SEED);
+		herbloreItems.add(ItemID.JANGERBERRY_SEED);
+		herbloreItems.add(ItemID.POISON_IVY_SEED);
+		herbloreItems.add(ItemID.BELLADONNA_SEED);
+	}
+
+	void createPrayerItems()
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: A static set would probably be better for this.

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java [hunk 5]
```diff
@@ -639,4 +639,65 @@ default Color bordercolor()
...
+	@ConfigItem(
+		keyName = "highlightPrayer",
+		name = "Highlight Prayer xp",
+		description = "Highlight Prayer xp related items.",
+		position = 53,
+		parent = "xpStub"
+	)
+	default boolean highlightPrayer()
+	{
+		return true;
```
Comment (Lucwousin) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java]: I think these should all be `false` by default

**Pull request merged**